src/components/PsaBanner/index.tsx

```diff
@@ -13,10 +13,10 @@ const WARNING_BANNER = 'WARNING_BANNER'
 const WebCoreBanner = (): ReactElement | null => {
   return (
     <>
-      ⚠️ Safe&apos;s new official URL is <a href={NEW_URL}>app.safe.global</a>, with a fully rebranded and refurbished
-      application.
+      ⚠️ This URL will be deprecated on February the 1st. ⚠️
```

Comment:
We should probably specify that it's not just the URL, but also the app. Wdyt?

Reply:
I think it's OK as is, because the app is still there, and it's 90% the same.
Feel free to propose a full alternative piece of text tho.

Reply:
The old app will be entirely removed though, right? Therefore, I'd extend it to say "This app and URL..." in line with the previous banner to be as clear as possible.

I will approve this nonetheless.

Reply:
Sounds good to me.